### ISSUE TITLE
t/spin/errors.t fails

### ISSUE THREAD

**Issue by eserte** [rra/docknot#6]
The test suite fails on some of my smoker systems:
```
#   Failed test 'errors are correct'
#   at t/spin/errors.t line 46.
#          got: 'errors.th:1: cannot find argument 2: Did not find opening bracket after prefix: "(?^:\G\s*)", detected at offset 2
# errors.th:3: invalid macro placeholder \2 (greater than 1)
# errors.th:5: invalid macro argument count for \badcount
# errors.th:9: unknown variable \=UNKNOWN
# errors.th:11: unknown command or macro \unknown
# errors.th:14: space in anchor "#foo bar"
# errors.th:15: no package release information available
# errors.th:16: no sitemap file found
# errors.th:17: no package version information available
# errors.th:17: cannot stat file nonexistent-file
# '
#     expected: 'errors.th:1: cannot find argument 2: Did not find opening bracket after prefix: "\s*", detected at offset 2
# errors.th:3: invalid macro placeholder \2 (greater than 1)
# errors.th:5: invalid macro argument count for \badcount
# errors.th:9: unknown variable \=UNKNOWN
# errors.th:11: unknown command or macro \unknown
# errors.th:14: space in anchor "#foo bar"
# errors.th:15: no package release information available
# errors.th:16: no sitemap file found
# errors.th:17: no package version information available
# errors.th:17: cannot stat file nonexistent-file
# '
# Looks like you failed 1 test of 2.
t/spin/errors.t ............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 
```

**Comment by rra** [comment]
This was fixed in b5613a5ed5873623fec56ff7f909f91800405be8 but that hasn't made it into a release. I need to push out a new release and then it should be possible to close this issue.

**Comment by rra** [comment]
Fixed in v8.0.0, now released.
